lib/Elastica/IndexTemplate.php

```diff
@@ -0,0 +1,118 @@
+<?php
+namespace Elastica;
+
+use Elastica\Exception\InvalidException;
+
+/**
+ * Elastica index template object.
+ *
+ * @author Dmitry Balabka <dmitry.balabka@gmail.com>
+ */
+class IndexTemplate
+{
+    /**
+     * Index template name.
+     *
+     * @var string Index pattern
+     */
+    protected $name = '';
```

Comment:
Yep, we take the SHOULD part here ;-) I fixed it and pushed it to master. Thanks for the contribution.